model/schema/variantDTO.yaml

```diff
@@ -36,33 +36,33 @@ emit_prefixes:
 ## DTO CLASSES
 ## ------------
 classes:
-
   VariantDTO:
     is_a: GenomicEntityDTO
     description: >-
       Ingest class for an Allele object
     slots:
-      - variant_type
-      - related_notes
-      - source_general_consequence
-      - variant_status
+      - variant_type_curie
+      - note_dtos
+      - source_general_consequence_curie
+      - variant_status_name
+
+  VariantLocationAssociationDTO:
+    abstract: true
+    is_a: GenomicLocationAssociationDTO
+    slots:
+      - variant_curie
+      - chromosome_curie
       - reference_sequence
       - variant_sequence
+      - consequence_curie
+      - curated_consequence_curie
       - sequence_of_reference_accession_number
 
   VariantGenomicLocationAssociationDTO:
-    is_a: AuditedObjectDTO
+    is_a: VariantLocationAssociationDTO
     slots:
-      - variant_entity_curie
-      - relation_name
-      - chromosome_curie
-      - evidence_curies
-      - evidence_code_curie
-      - assembly_curie
-      - start
-      - end
-      - exon_number
-      - intron_number
+      - exon_number # should this be a property of the transcript location? different isoforms could have different values for this
+      - intron_number # should this be a property of the transcript location? different isoforms could have different values for this
```

Comment:
I agree that it is odd that these slots (exon_number and intron_number) are added to this class. There are no corresponding slots in the non-DTO model, as far as I can tell, and I agree it likely can't be unambiguous for genomic location unless specifying a particular transcript isoform, if there is more than one.